unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -1106,4 +1109,19 @@ private void SendEquipWearableAnalytic(WearableItem equippedWearable)
     }
 
     internal virtual IEmotesCustomizationComponentController CreateEmotesController() => new EmotesCustomizationComponentController();
+
+    public void LostFocus()
+    {
+        lostFocus = true;
+    }
```

Comment:
Can't we just update `lastTimeOwnedWearablesChecked` to avoid adding a new `lostFocus` field?

Also, I think we should do the same for the owned emotes.